hs-abci-examples/nameservice/src/Nameservice/Modules/Nameservice/Router.hs

```diff
@@ -20,6 +23,18 @@ router
 router (RoutedTx Tx{txMsg}) =
   let Msg{msgData} = txMsg
   in case msgData of
-       NSetName msg    -> setName msg
-       NBuyName msg    -> buyName msg
-       NDeleteName msg -> deleteName msg
+       NSetName msg    -> do
+         incCount "count_set"
+         observeHistogram "histogram_set" 0.0
```

Comment:
what are these uses of `observeHistogram` attempting to record?

Reply:
This is how you init a histogram. The value here (i.e., `0.0`) is just a dummy value.